dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/StartUpMetainformationPrinter.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022-2025 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.jar.Manifest;
+
+public class StartUpMetainformationPrinter {
```

Comment:
Should not `MetaInformation` be more readable instead of `Metainformation`?

Reply:
`Metainformation` is one word, isn't ?

Reply:
Hmm, I heard only `metadata` as a term.